README.Rmd

```diff
@@ -258,22 +257,38 @@ The function does not throw an error and inform that no duplicates were
 found.
 
 ```{r}
-check_duplicated_score(manually_matched)
+check_duplicated_relation(manually_matched)
 ```
 
-Now let us insert a duplicated match at row 2 : let us put TRUE instead
-of NA.
+Now let us insert duplicated matches for the company with the ids 1 and
+2.
 
 ```{r}
-manually_matched$accept_match[2] <- TRUE
+duplicate_in_loanbook <- manually_matched %>% 
+  dplyr::mutate(accept_match = dplyr::if_else(id %in% c(1, 2), TRUE, accept_match))
 
-knitr::kable(head(manually_matched))
+knitr::kable(duplicate_in_loanbook %>% dplyr::filter(id %in% c(1, 2)))
 ```
 
 The function then abort and throws an error with the lines of the
 duplicated rows.
 
 ```{r}
 # un-comment this line to have the error
-# check_duplicated_score(manually_matched)
+# check_duplicated_relation(duplicate_in_loanbook)
+```
+
+Also, an error is thrown if we insert a duplicate match of a company
+from the tilt db to the loanbook.
+
+```{r}
+duplicate_tilt_id <- demo_matched %>% 
```

Comment:
I think I would probably instead add a row, exactly the same as id == 3 (in the loan book) but with a new id (in the loanbook).
I think it would illustrate this edgecase better, since an id_tilt should be unique per zip code, which is not the case here I think.
Also, it will show the user that he/she should select one of the two rows